static/app/views/stories/index.tsx

```diff
@@ -78,3 +91,6 @@ const Main = styled(VerticalScroll)`
 
   position: relative;
 `;
+
+const StyledInput = styled(Input)`
+  margin-bottom: ${space(2)};`;
```

Comment:
grumble

Reply:
it wouldn't let me :(

Reply:
ya, i know we have an issue with these blocks now :(